assignments/arrays.js

```diff
@@ -71,20 +71,32 @@ console.log(`Car 33 is a ` + inventory[32].car_year +" " + inventory[32].car_mak
 let lastCar = 0;
 lastCar = inventory;
 for (i = 0; i < lastCar.length-1; i++ ){
-   //console.log(inventory.car_make);
-
+   //
 }
 console.log(`Make:`+ " " + inventory[i].car_make + " "+ `Model:` + " " + inventory[i].car_model);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+for (let i = 0; i < inventory.length; i++){
+    let model = inventory[i].car_model;
+    carModels.push(model);
+    carModels.sort();
```

Comment:
using the sort method here does work. but you don't want to invoke a sort method inside of a loop. by doing this the sort will invoke in every iteration of the loop which will do a lot more work then we want to do and can impact performance.

the best practice in this case is to allow the loop to finish and then invoke the sort on the `carModels` below the for loop